### ISSUE TITLE
Percolator only crashes, Qvality doesn't work

### ISSUE THREAD

**Issue by NameyMcNamo** [ursgal/ursgal#64]
Hello,

Thanks for making ursgal.

I'm trying to use just the basic search.py example and it seems the output of qvality will only be "1" for PEP, and "-1.#IND" for q-value regardless of any input.

Secondly, I can't get percolator to work on any data sets. I'd found a previous issue with this, Issue#13 ( https://github.com/ursgal/ursgal/issues/13 ), but the only inference was that the data set was not large enough. I've had this crash on both small and large data sets alike with no success whatsoever.

The largest data set I've tried is a proteomic experiment available on PRIDE, q01511.raw of:
https://www.ebi.ac.uk/pride/archive/projects/PXD008222/files
that was searched with the canonical & isoform proteom from uniprot:
http://www.uniprot.org/proteomes/UP000005640

I believe this would be large enough to perform validation on based on the number of peptides identified in the publication.

The version I'm using was cloned yesterday. I'm really interested in getting this workflow up and running, is there anything I could do to alleviate this problem?

Output from large data set crash and the small data set crash are the same:
```
percolator_2_08 crashed!

  The executable
    ursgal\resources\win32\64bit\percolator_2_08\percolator.exe
  terminated with Error code 3221226356 .
  Inspect the printouts above for possible causes and verify that all input files are valid.
```

**Comment by StSchulze** [comment]
Hi NameyMcNamo,

thanks for using Ursgal ;-)

I am not sure which example script with validation you mean, the simple_example_search.py script doesn't include a validation step, but e.g. the xtandem_version_comparison.py or the do_it_all_folder_wide.py does. The first one should work without any additional input. But it doesn't for you?

The PRIDE dataset looks more than large enough, yes. However, did you generate a target-decoy database after downloading the human proteome? If not, then this could be you problem. There is an example script for that: target_decoy_generation_example.py (although I would suggest to use 'shuffle_peptide' isntead of 'reverse_protein')

If this the above doesn't work and if the missing decoy is not the problem, then could you include the script that you are using and ideally more of the output message before the crash report (i.e. the output message of Percolator that you can also see in #13)?

**Comment by NameyMcNamo** [comment]
Oh, I had been doing that wrong hah. The do_it_all folder_wide worked after generating decoys.

Sorry, sometimes I swear I belong in a special ed class.
Cheers :)

**Comment by StSchulze** [comment]
No problem, let us know if you encounter any other problems ;)

**Comment by NameyMcNamo** [comment]
I do have a quick question, where can one find the MSAmanda engines to download and put into the resources folder?

**Comment by StSchulze** [comment]
You can download the latest MSAmanda version here: http://ms.imp.ac.at/index.php?action=ms-amanda
(see the standalone versions somewhere at the bottom)
For older versions you probably need to contact them directly.
We have implemented the new version in Ursgal v0.6.0 (see pull request #57), which will probably be merged in the next days, but you can also directly pull this branch. In general, there are some bigger changes in v0.6.0, so if you are starting with Ursgal right now, I would suggest to directly use v0.6.0 to avoid compatibility problems. Sorry, I might should have mentioned this earlier ...

**Comment by NameyMcNamo** [comment]
That's great, I'll jump into the newer version when it's ready. Thanks for letting me know.
